cmd/helm/downloader/chart_downloader.go

```diff
@@ -125,11 +137,11 @@ func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *proven
 //
 // A version is a SemVer string (1.2.3-beta.1+f334a6789).
 //
-// 	- For fully qualified URLs, the version will be ignored (since URLs aren't versioned)
+//	- For fully qualified URLs, the version will be ignored (since URLs aren't versioned)
 //	- For a chart reference
 //		* If version is non-empty, this will return the URL for that version
 //		* If version is empty, this will return the URL for the latest version
-// 		* If no version can be found, an error is returned
+//		* If no version can be found, an error is returned
 func (c *ChartDownloader) ResolveChartVersion(ref, version string) (*url.URL, *repo.ChartRepository, error) {
```

Comment:
@adamreese I feel like maybe we should change the return value from a `*repo.ChartRepository` to a `repo.Getter`, since what this function is really intended to to do is resolve a reference to a downloadable unit.

The main issue with the bug I am resolving is that at some point someone assumed that every chart URL _must_ be in a known repo. This method signature matches that assumption, but really it should be saying "here is the URL and a client to use to download that URL".

Thoughts? Or should I just leave it?